mirage/factories/utils.ts

```diff
@@ -114,3 +141,39 @@ export function createRegistrationMetadata(
     }
     return registrationMetadata;
 }
+
+/**
+ * Create registrationResponse according to schemaBlocks
+ */
+export function createRegistrationResponses(
+    registrationSchema: ModelInstance<MirageRegistrationSchema>,
+) {
+    const schemaBlocks = registrationSchema.schemaBlocks!;
+    const schemaBlockGroups = getSchemaBlockGroups(schemaBlocks);
+    const registrationResponses = {} as RegistrationResponses;
+    for (const group of schemaBlockGroups) {
```

Comment:
I think this would be more cleanly implemented as a .filter().map():
```ts
return schemaBlockGroups.filter(group => Boolean(group.registrationResponseKey)).map(group => {
    const { groupType, registrationResponseKey } = group;
    switch (groupType) {
    case 'long-text-input':
        return faker.lorem.paragraph();
```